source/administration/changelog.md

```diff
@@ -5,10 +5,217 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
 Also see [changelog in progress](http://bit.ly/2nK3cVf) for the next release.
 
 ## Release v4.2.0
-Scheduled release date: 2017-09-16
+Release date: 2017-09-16
```

Comment:
Release is on Friday, September 15th.  Are we just going to leave this at the 16th for continuity?

Reply:
I think it's better we keep it as the 16th, since we say "release on the 16th of every month" in a few places.

Reply:
Got it, agree :)  Thanks Jason!